deployment/common/changeset/internal/mcms_test.go

```diff
@@ -36,9 +37,9 @@ func TestDeployMCMSWithTimelockContracts(t *testing.T) {
 	_, err := internal.DeployMCMSWithTimelockContracts(lggr,
 		chains[chainsel.TEST_90000001.Selector],
 		ab, types.MCMSWithTimelockConfig{
-			Canceller:        changeset.SingleGroupMCMS(t),
-			Bypasser:         changeset.SingleGroupMCMS(t),
-			Proposer:         changeset.SingleGroupMCMS(t),
+			Canceller:        proposalutils.SingleGroupMCMS(t),
+			Bypasser:         proposalutils.SingleGroupMCMS(t),
+			Proposer:         proposalutils.SingleGroupMCMS(t),
 			TimelockMinDelay: big.NewInt(0),
```

Comment:
Why not just `proposalutils.SingleGroupTimelockConfig(t)` ?

Reply:
oversight. thanks